src/make.ts

```diff
@@ -49,19 +49,11 @@ export async function createFiles({contract, frontend, tests, projectPath, verbo
   fs.mkdirSync(targetTestDir, { recursive: true });
 
   // copy tests - shared files
-  const sourceTestSharedDir = path.resolve(`${rootDir}/integration-tests/shared/${tests}-tests`);
+  const sourceTestSharedDir = path.resolve(`${rootDir}/integration-tests/${tests}-tests`);
   await copyDir(sourceTestSharedDir, targetTestDir, {
     verbose,
     skip: skip.map(f => path.join(sourceTestSharedDir, f))
   });
-  // copy tests - overrides files
```

Comment:
@maxhr JS does not need to be initialised anymore, for which we can use the same tests in all examples